docs/destinations/warehouses-and-lakes/rdb/transforming-enriched-data/reusable/stream-transformer-common/_index.mdx

```diff
@@ -46,3 +46,7 @@ import Link from '@docusaurus/Link';
     <td><code>monitoring.sentry.dsn</code></td>
     <td>Optional. For tracking runtime exceptions.</td>
 </tr>
+<tr>
```

Comment:
Why add this to both batch-only and stream-common?

Reply:
Different defaults, batch -`false`, stream - `true`.